registry/modules/aspect_rules_py/enf-1.5.1/patches/aspect_rules_py.patch

```
@@ -0,0 +1,253 @@
+-        major, minor, micro = ctx.attr._interpreter_version_flag[BuildSettingInfo].value.split(".")
++        major, minor, micro = interpreter_version.split(".")
+         interpreter_version_info = struct(
+             major = major,
+             minor = minor,
+@@ -111,7 +100,23 @@ def _resolve_toolchain(ctx):
+         flags = _INTERPRETER_FLAGS,
+     )
+ 
++
++def _resolve_toolchain(ctx):
++    """Resolves the Python toolchain to a simple struct.
++
++    Args:
++        ctx: Bazel rule context.
++
++    Returns:
++        Struct describing the Python toolchain to use.
++    """
++    return _resolve_toolchain_from_info(
++        ctx.toolchains[PY_TOOLCHAIN],
++        ctx.attr._interpreter_version_flag[BuildSettingInfo].value,
++    )
++
+ semantics = struct(
+     interpreter_flags = _INTERPRETER_FLAGS,
+     resolve_toolchain = _resolve_toolchain,
++    resolve_toolchain_from_info = _resolve_toolchain_from_info,
+ )
+diff --git py/tools/pex/main.py py/tools/pex/main.py
+index 5c699fd..2fcdbd1 100644
+--- py/tools/pex/main.py
++++ py/tools/pex/main.py
+@@ -159,16 +159,19 @@ pex_info.interpreter_constraints = [
+ ]
+ 
+ for dep in options.dependencies:
+-    dist = Distribution.load(dep)
+-
+-    # TODO: explain which level of inferno is this!
+-    key = "%s-%s" % (dist.key, dist.version)
+-    dist_hash = pex_builder._add_dist(
+-        path= dist.location,
+-        dist_name = key
+-    )
+-    pex_info.add_distribution(key, dist_hash)
+-    pex_builder.add_requirement(dist.as_requirement())
++    try:
++        dist = Distribution.load(dep)
++
++        # TODO: explain which level of inferno is this!
++        key = "%s-%s" % (dist.key, dist.version)
++        dist_hash = pex_builder._add_dist(
++            path= dist.location,
++            dist_name = key
++        )
++        pex_info.add_distribution(key, dist_hash)
++        pex_builder.add_requirement(dist.as_requirement())
++    except:
++        print("Can't parse dist in " + dep) 
```

Does this swallow an error that should be propagated up?